### PR TITLE
Add spectral-cube to registry

### PR DESCRIPTION
**Pull request by keflavich**
@eteq as requested

### CODE REVIEW

File: affiliated/registry.json [hunk 1]
```diff
@@ -161,6 +161,15 @@
             "home_url": "http://gwcs.readthedocs.org/en/latest/",
             "repo_url": "https://github.com/spacetelescope/gwcs.git",
             "pypi_name": "gwcs"
+        },
+        {
+            "name": "spectral-cube",
+            "maintainer": "Adam Ginsburg",
+            "provisional": false,
+            "stable": true,
```
Comment (eteq) [inline affiliated/registry.json]: nominally this is your call, but are you sure you want to call it "stable"?  That is, I presume you've still got a lot of work you're planning on doing?

Comment (keflavich) [inline affiliated/registry.json]: It's pretty stable, honestly.  We're not planning on adding many features except perhaps more IO

Comment (astrofrog) [inline affiliated/registry.json]: I've been understanding 'stable' to mean 'there is a stable release' - which has implications on e.g. if the package can be pip-installed or not.

**Comment by astrofrog** [comment]
@eteq - can we merge this?

**Comment by eteq** [comment]
@astrofrog - No, you can't... because I did just now ;)

**Pull request merged**